scripts/grpc.mjs

```diff
@@ -0,0 +1,106 @@
+import { execFile } from 'child_process';
+import { gray, green, red } from 'colorette';
+import { mkdir, readdir, rm, writeFile } from 'fs/promises';
+import { dirname, join, sep } from 'path';
+import { fileURLToPath } from 'url';
+import { promisify } from 'util';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
```

Comment:
Remove and replace with `new URL` stuff to get all the directories and write all the files.

Reply:
This script depends on `execFile`, which does not support `URL`s:

![image](https://user-images.githubusercontent.com/24852502/114153094-44aa9d00-991f-11eb-8f3c-9ec492de90ea.png)